### DISCUSSION TITLE
Assumption about the transfer to NeoForge

### DISCUSSION THREAD

**Discussion by ghost** [ldtteam/minecolonies#10192] [Q&A] [answered]
I'm just wondering, I recently noticed that all the additional mods for minecolonies have been updated to version 1.21.1 not on Forge, but on NeoForge, and I wanted to ask if minecolonies will be migrated to NeoForge or will it also remain on Forge?

**Comment by Raycoms** [comment, marked as answer]
Because of the amount of work it takes to maintain two versions, we'll be only on Neoforge for now.

**Comment by ghost** [comment]
thanks
